app/views/DR/Maps/index.js

```diff
@@ -18,11 +18,17 @@ const index = () => {
         }}
         swipeEnabled={false}>
         <Tab.Screen
-          name={languages.t('navigation.hospitals_maps')}
+          name={'Hospitals'}
```

Comment:
```suggestion
          name="Hospital"
```

Reply:
Why not in plural?

Reply:
Why not in plural? And why in double quote?

Reply:
It's still in plural, the suggestion is mostly about the brackets "{ }"